pages/feedback.js

```diff
@@ -55,6 +56,13 @@ export class Feedback extends Component {
             options={question.multiple_choice_options}
             store={store}
           />
+          <TextArea
+            name="group1"
+            maxLength={t("feedback.text_area_char_limit")}
```

Comment:
I think we should hard code this rather than get the value from airtable. If we sent feedback to a database with a char limit on this field, it could be bad to have it changed inadvertently in airtable.